scripts/ci/build.sh

```diff
@@ -2,17 +2,26 @@
 set -ex
 
 ROOTDIR="$(dirname "$0")/../.."
+# shellcheck source=scripts/common.sh
+source "${ROOTDIR}/scripts/common.sh"
+
+prerelease_source="$1"
```

Comment:
And then you can move the default here:
```suggestion
prerelease_source="${1:-ci}"
```